ampersand-select-view.js

```diff
@@ -47,9 +47,7 @@ module.exports = View.extend({
         }
 
         this.el = opts.el;
-        if (opts.label === undefined) {
-            this.label = this.name;
-        } else {
+        if (opts.label !== undefined) {
```

Comment:
This `if` could probably be removed since we check for `this.label` in the render function now.